electroncash/commands.py

```diff
@@ -175,6 +178,59 @@ def commands(self):
         """List of commands"""
         return ' '.join(sorted(known_commands.keys()))
 
+    @command('n')
+    def getinfo(self):
+        """ network info """
+        net_params = self.network.get_parameters()
+        response = {
+            'path': self.network.config.path,
+            'server': net_params[0],
+            'blockchain_height': self.network.get_local_height(),
+            'server_height': self.network.get_server_height(),
+            'spv_nodes': len(self.network.get_interfaces()),
+            'connected': self.network.is_connected(),
+            'auto_connect': net_params[4],
+            'version': PACKAGE_VERSION,
+            'wallets': {k: w.is_up_to_date()
+                        for k, w in self.daemon.wallets.items()},
+            'fee_per_kb': self.config.fee_per_kb(),
+        }
+        return response
+
+    @command('n')
+    def stop(self):
+        """Stop daemon"""
+        self.daemon.stop()
+        return "Daemon stopped"
+
+    @command('n')
+    def list_wallets(self):
+        """List wallets open in daemon"""
+        return [{'path':k, 'synchronized':w.is_up_to_date()} for k, w in self.daemon.wallets.items()]
+
+    @command('n')
+    def load_wallet(self):
+        """Open wallet in daemon"""
+        path = self.config.get_wallet_path()
+        wallet = self.daemon.load_wallet(path, self.config.get('password'))
+        if wallet is not None:
+            self.wallet = wallet
+            run_hook('load_wallet', wallet, None)
```

Comment:
Upon a little inspection this seems a bit problematic since the hook is not executed in the main thread.